### PR TITLE
i18n(es): Translated `invalid-get-static-paths-entry` & updated `error-reference.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `invalid-get-static-paths-entry` & updated `error-reference.mdx` based on this commit fa3eab87eec4a0a7c33699f17f69148084872f7a

### CODE REVIEW

File: src/content/docs/es/reference/errors/invalid-get-static-paths-entry.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Invalid entry inside getStaticPath's return value
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+
+> **InvalidGetStaticPathsEntry**: Entrada inválida retornada por getStaticPaths. Esperando un objecto, obtuve `ENTRY_TYPE`
```
Comment (Waxer59) [inline src/content/docs/es/reference/errors/invalid-get-static-paths-entry.mdx:8]:
```suggestion
> **InvalidGetStaticPathsEntry**: Entrada inválida retornada por getStaticPaths. Se esperaba un objecto, se obtuvo `ENTRY_TYPE`
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Comment by dreyfus92** [comment]
Gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work 🚀

**Pull request merged**